### PR TITLE
Make appconfig IT to be independent

### PR DESCRIPTION
**Pull request by Netyyyy**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

Adjust the position of appconfiguration-config IT to make it an independent part 

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by Netyyyy** [comment]
@mrm9084 please help take a look, if it is ok, I will sync to springboot3 as well, thanks.

File: sdk/spring/tests.yml [hunk 1]
```diff
@@ -28,14 +27,32 @@ stages:
...
+          SubscriptionConfiguration: $(sub-config-cn-test-resources)
+      TestResourceDirectories:
+        - spring/spring-cloud-azure-integration-test-appconfiguration-config
+      Artifacts:
+        - name: spring-cloud-azure-integration-test-appconfiguration-config
+          groupId: com.azure.spring
+          safeName: springcloudazureintegrationtestappconfigurationconfig
+      TimeoutInMinutes: 60
+      ServiceDirectory: spring
+      TestName: Appconfiguration_Config_IT
```
Comment (saragluna) [inline sdk/spring/tests.yml:54]: Spring_App_Config_IT

**Review by saragluna — APPROVED** [review]
LGTM, but we need to wait for Matt to sign off.

**Review by mrm9084 — APPROVED** [review]
I assume we will also make the change in the spring-boot-3 branch

**Pull request merged**